### DISCUSSION TITLE
Rand3DElasticDict

### DISCUSSION THREAD

**Discussion by lyuxiaobai** [Project-MONAI/MONAI#1483] [Q&A]
For segmentation tasks, are there any suggestions for the values ​​of sigma and magnitude_range of Rand3DElasticDict? If I pass the image and real annotations into this method at the same time, how should the mode be written? I want to use bilinear for images and nearest for real labeling, but I don’t know how to achieve this

**Comment by lyuxiaobai** [comment]
![Uploading image.png…]()

**Comment by lyuxiaobai** [comment]
train_transforms = Compose([
    LoadNiftid(keys={'image','label'}),
#     AddChanneld(keys={'image','label'}),
    ScaleIntensityd(keys={'image'}),
    Rand3DElasticDict(keys={'image','label'},sigma_range=(15,25),magnitude_range=(0,5),prob=0.4,mode='bilinear,nearest'),
    RandFlipDict(keys={'image','label'}),
    ToTensor()
])
代码如上

**Comment by Nic-Ma** [comment]
Hi @lyuxiaobai ,

Thanks for your interest in MONAI.
I saw you already set `mode='bilinear, nearest'` in your code for `image` and `label`.
So could you please share more details about your question?

Thanks.
